### ISSUE TITLE
Issue with module loading while using Next.js and turbopack

### ISSUE THREAD

**Issue by mantidfly** [chakra-ui/panda#1172]
### Subject

Installation instructions for Next.js

### Description

Hi! Thanks for all the work you put in this library. ❤️ 

I tried to play with Panda as potential replacement for Stitches and by following the installation steps for Next.js I've encountered following blocking issue:
```
error - [resolve] [project]/app/page.tsx  Error resolving EcmaScript Modules request
  unable to resolve module "@" with subpath "/styled-system/css"
```

I spend some time digging it up and it seems that fresh Next.js + turbopack project doesn't work well with `mjs` extension. To make it work I had to add following setting to `panda.config.ts`
```
outExtension: 'js',
```

Here you can reproduce the issue: https://codesandbox.io/p/sandbox/cranky-jang-m2lmns?file=%2Fpackage.json%3A11%2C5

If you don't replace `mjs` with `js` you have to remove `--turbo` from package.json.

Maybe it would be worth to add as a disclaimer. Cheers!

**Comment by anubra266** [comment]
@mantidfly Turbopack is a beta project, we can't take the burden of keeping up with the bugs and frequent changes that will happen till it gets to alpha. But we've documented this config key.
You can also pass it as a flag when initializing panda in your future turbopack projects

**Comment by dbstratta** [comment]
It was fixed in Next 14.0.4.
